tests/end-to-end/bivariate/test_base.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pytest
+
+from copulas.bivariate import Bivariate
+
+
+class TestBivariate():
+
+    @pytest.mark.parametrize('model', Bivariate._get_subclasses())
```

Comment:
We should use the `subclasses()` method instead of `_get_subclasses`, since this one is intended to be used privately only, and they return the same information.